contribs/gmf/src/controllers/abstractmobile.js

```diff
@@ -95,14 +112,12 @@ gmf.AbstractMobileController = function(ngeoFeatureOverlayMgr, serverVars) {
    * @export
    */
   this.map = new ol.Map({
-    layers: [
-      new ol.layer.Tile({
-        source: new ol.source.OSM()
-      })
-    ],
+    layers: [],
     view: new ol.View({
-      center: [0, 0],
-      zoom: 2
+      center: [632464, 185457],
+      projection: ol.proj.get('epsg:21781'),
+      minZoom: 3,
+      zoom: 3
```

Comment:
@sbrunner we will need to use contant for view settings as well (projection, center, zoom)